README.md

```diff
@@ -1,2 +1,205 @@
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://html.org/">html</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://css.com/">CSS</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Design**
+- **Form**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+<!--
+Example command:
+
+```sh
+ gem install rails
+```
+ -->
+
+Example commands:
+
+```sh
+  cd my-folder
+  git clone https://github.com/omer-creator/survey-form-review.git
+  npm i
+```
+
+Linters command to check html:
+
+```sh
+  npx hint .
+```
+
+Linters command to check css:
+
+```sh
+  npx stylelint "**/*.{css,scss}"
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+<!--
+Example command:
+
+```sh
+  rails server
+```
+--->
+
+### Run tests
+
+To run tests, run the following command:
+
+<!--
+Example command:
+
+```sh
+  bin/rails test test/models/article_test.rb
+```
+--->
+
+### Deployment
+
+You can deploy this project using:
+
+<!--
+Example:
+
+```sh
+
+```
+ -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> Muhammad Omer Afzal
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/omer-creator)
+- Twitter: [@twitterhandle](https://www.linkedin.com/in/muhammad-omer-afzal-67ab17250/)
+- LinkedIn: [LinkedIn](https://twitter.com/Micronaut_Omer)
+
+> victor Akawara
+
+👤 **Author2**
+
+- GitHub: [@githubhandle](https://github.com/victormor)
+- Twitter: [@twitterhandle](https://www.linkedin.com/victormor)
+- LinkedIn: [LinkedIn](https://twitter.com/victormore)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+
+- [ ] **Enhancing Css classes**
+- [ ] **Use css minifier to get one css file**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+
+If you like this project...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+> Give credit to everyone who inspired your codebase.
+
+I would like to thank...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
Great job adding a perfect readme file. I suggest you also add the license to your readme file. This will allow others to understand what they can do with your project.